apps/examples/testcase/le_tc/drivers/tc_dev_null.c

```diff
@@ -0,0 +1,75 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_dev_null.c
+/// @brief Test Case Example for dev_null driver
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include "tc_internal.h"
+#ifndef CONFIG_DISABLE_POLL
+#include <poll.h>
+#endif
+
+/****************************************************************************
+ * Name: dev_null driver test
+ ****************************************************************************/
+/**
+* @fn                   :dev_null_main
+* @brief                :Test the dev_null driver. Since the driver is extremely small, all tests are done here
+* @scenario             :Test the dev_null driver
+* API's covered         :open, read, write, poll
+* Preconditions         :none
+* Postconditions        :none
+* @return               :void
+*/
+
+void dev_null_main(void)
+{
+	int fd = 0;
+	int ret = 0;
+
+	fd = open("/dev/null", O_RDWR);
+	TC_ASSERT_GT("dev_null_tc open", fd, 0);
+
+	ret = read(fd, NULL, 0);
+	TC_ASSERT_EQ_CLEANUP("dev_null_tc read", ret, 0, close(fd));
```

Comment:
in TC_ASSERT_EQ_CLEANUP("XXX", );
XXX will be API name.

Reply:
Done